test/foundry/MockAgEUR.test.sol

```diff
@@ -23,11 +23,6 @@ contract MockAgEURTest is Test {
         token = new MockAgEUR();
     }
 
-    function destroy() public returns (uint256) {
```

Comment:
why deleted?

Reply:
because self destroy isn't really future proof as it's still intended to deprecate this opcode